packages/core/src/asciidoctor-core-api.js

```diff
@@ -2041,6 +2044,24 @@ MemoryLogger.prototype.getMessages = function () {
   return result
 }
 
+var Logging = Opal.const_get_qualified(Opal.Asciidoctor, 'Logging', true)
+
+Opal.Asciidoctor.Logging = Logging
+
+Logging.getLogger = function () {
+  return LoggerManager.$logger()
+}
+
+Logging.messageWithContext = function (text, context) {
```

Comment:
Since this isn't a DSL in JavaScript, perhaps we should go with a more method-like name such as `createMessageWithContext` or even just `createLogMessage` (implying that it is adding context).

Reply:
:+1: 
Indeed, I've settled on `createLogMessage`

Reply:
I like it.